assets/prototype/application/constants/time.ts

```diff
@@ -0,0 +1,9 @@
+enum TIME {
```

Comment:
I've made this uppercase to make it consistent.

Reply:
yup... smart 👍